catalyst_voices/packages/internal/catalyst_voices_services/lib/src/api_models/overriden_models.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:catalyst_voices_services/generated/catalyst_gateway/vit.models.swagger.dart';
```

Comment:
It would be nice if vitss was not under `catalyst)gateway` because its not part of catalyst gateway.
Just so the distinction is clearer.
Perhaps in `package:catalyst_voices_services/generated/vitss/*` instead?

Reply:
yeah, wanted to do it too but [swagger_dart_code_generator](https://pub.dev/packages/swagger_dart_code_generator) allows only one `input_folder`/`output_folder` folder. 

I was thinking about renaming `catalyst_gateway` to `api` because later generated classes have descriptive names anyways.

so from 
```text
├── generated
│   └── catalyst_gateway
│       ├── cat_gateway_api.enums.swagger.dart
│       ├── cat_gateway_api.models.swagger.dart
│       ├── cat_gateway_api.models.swagger.g.dart
│       ├── cat_gateway_api.swagger.chopper.dart
│       ├── cat_gateway_api.swagger.dart
│       ├── client_index.dart
│       ├── client_mapping.dart
│       ├── vit.enums.swagger.dart
│       ├── vit.models.swagger.dart
│       ├── vit.models.swagger.g.dart
│       ├── vit.swagger.chopper.dart
│       └── vit.swagger.dart
```

to 
```text
├── generated
│   └── api
│       ├── cat_gateway_api.enums.swagger.dart
│       ├── cat_gateway_api.models.swagger.dart
│       ├── cat_gateway_api.models.swagger.g.dart
│       ├── cat_gateway_api.swagger.chopper.dart
│       ├── cat_gateway_api.swagger.dart
│       ├── client_index.dart
│       ├── client_mapping.dart
│       ├── vit.enums.swagger.dart
│       ├── vit.models.swagger.dart
│       ├── vit.models.swagger.g.dart
│       ├── vit.swagger.chopper.dart
│       └── vit.swagger.dart
```

does it make sense?

Reply:
maybe even rename `cat_gateway_api` to just `cat_gateway`